ModelBundle/Migrations/MongoDB/Version20150722104732.php

```diff
@@ -24,7 +24,7 @@ public function getDescription()
     public function up(Database $db)
     {
         $this->executeScript($db, 'create_duplicate_node_20150722104732.js');
-
+        $this->executeScript($db, 'create_select_enumeration_20150722104732.js');
```

Comment:
no, if you want to add a new proc stored, you need to add a new migration.

If we run again migration migrate, this migration will not be executed again